sdk/core/azure-core-serializer-json-gson/src/main/java/module-info.java

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+module com.azure.core.serializer.json.gson {
+    requires transitive com.azure.core;
+    requires transitive gson;
```

Comment:
This should export `com.azure.core.serializer.json.gson`.

Reply:
Added